tree/ntuple/v7/test/ntuple_types.cxx

```diff
@@ -1935,7 +1935,7 @@ TEST(RNTuple, TClass)
          auto viewKlass = ntuple->GetView<DerivedA>("klass");
          FAIL() << "GetView<a_base_class_of_T> should throw";
       } catch (const RException& err) {
-         EXPECT_THAT(err.what(), testing::HasSubstr("type mismatch for field klass"));
+         EXPECT_THAT(err.what(), testing::HasSubstr("No on-disk field information"));
```

Comment:
Hm, alternatively we could restrict casting to non-class types and keep the current error message. What do you think?

Reply:
I think it makes sense as is. Otherwise, we'd also need to handle vector of class etc.